packages/design-system/src/components/Icon/Icon.module.scss

```diff
@@ -0,0 +1,57 @@
+@use '~@talend/design-tokens/lib/tokens' as tokens;
+
+.border {
+	circle,
+	path,
+	polygon,
+	polyline {
+        // only if border
+	    transform: translate(25%, 25%);;
+	}
+    :global(.ti-border) {
+        stroke: currentColor; fill: none; transform: none;
+    }
+}
+
+.svg {
+    fill: currentColor;
+	width: tokens.$coral-sizing-xxs;
+	height: tokens.$coral-sizing-xxs;
+	transform-origin: center;
+
+
+
+	&.spin {
+		animation-name: svg-spin;
+		animation-duration: 2s;
+		animation-iteration-count: infinite;
+		animation-timing-function: linear;
+	}
+	&.rotate-45 {
```

Comment:
This is the legacy icon component right? 

If you guys still want these, then they should be wrapped in a :global declaration. Otherwise you can't get them through the opened `className` props, CSS module would not know what's what by then.

Reply:
True this is legacy, and it could be done at the app level if needed.
But even if it is not used it will create a breaking change :thinking: 
I would go one step at a time and prefer to keep it as it is, just drop styled components first